blocks/embed/lite-vimeo-embed/lite-vimeo-embed.js

```diff
@@ -32,40 +33,47 @@ class LiteVimeo extends (globalThis.HTMLElement ?? class {}) {
     if (devicePixelRatio >= 2) devicePixelRatio *= .75;
     width = Math.round(width * devicePixelRatio);
     height = Math.round(height * devicePixelRatio);
+    if(imageUrl){
+      this.style.backgroundImage = `url("${imageUrl}_${width}x${height}")`;
+    }
+
+    let playBtnEl = this.querySelector('.ltv-playbtn');
+      // A label for the button takes priority over a [playlabel] attribute on the custom-element
+      this.playLabel = (playBtnEl && playBtnEl.textContent.trim()) || this.getAttribute('playlabel') || 'Play video';
+
+      if (!playBtnEl) {
+      playBtnEl = document.createElement('button');
```

Comment:
```suggestion
        playBtnEl = createTag('button', {aria-label: `${this.playLabel}`, class: 'ltv-playbtn' } );
```
FWIW, we could reduce the line numbers 45-48 w/ this.. would need to also `import { createTag } from /utils/utils.js` etc. 

Same as below on line 66 w/ the `H5` etc.